src/components/views/beacon/BeaconViewDialog.tsx

```diff
@@ -42,7 +42,7 @@ interface IProps extends IDialogProps {
     focusBeacon?: Beacon;
 }
 
-const getBoundsCenter = (bounds: Bounds): string => {
+const getBoundsCenter = (bounds: Bounds): string | undefined => {
     if (!bounds) {
```

Comment:
bounds is also tested for falsiness, yet the signature says it will never be falsey.